Auth/Auth.Application/Abstractions/Interfaces/Repositories/IUserRepository.cs

```diff
@@ -5,11 +5,12 @@ namespace Auth.Application.Abstractions.Interfaces.Repositories
 {
     public interface IUserRepository
```

Comment:
Because IUserRepository contains user related methods, it is not a base repo. RepositoryBase is. It inherits IRepositoryBase. And then UserRepository inherits RepositoryBase and IUserRepository. Isn't is right?